internal/pkg/agent/application/monitoring/server.go

```diff
@@ -54,6 +59,13 @@ func exposeMetricsEndpoint(
 	statsHandler := statsHandler(ns("stats"))
 	r.Handle("/stats", createHandler(statsHandler))
 
+	if enableProcessStats {
+		r.Handle("/processes", createHandler(processesHandler(coord)))
+		r.Handle("/processes/{processID}", createHandler(processHandler(coord, statsHandler)))
+		r.Handle("/processes/{processID}/", createHandler(processHandler(coord, statsHandler)))
+		r.Handle("/processes/{processID}/{beatsPath}", createHandler(processHandler(coord, statsHandler)))
```

Comment:
What is `{beatsPath}` here? I don't see it used in the handler.

Reply:
removed